rmshit.py

```diff
@@ -61,6 +60,8 @@
 - ~/.local/share/Trash/    # VSCode puts deleted files here
 """
 
+def get_size(folder):
+    return sum(p.stat().st_size for p in Path(folder).rglob('*'))
```

Comment:
Oh, this does not work for regular files. Should be like this:

```suggestion
def get_size(path):
    if path.is_dir():
        return sum(p.stat().st_size for p in Path(folder).rglob('*'))
    return path.stat().st_size
```